addons/sourcemod/scripting/shop.sp

```diff
@@ -1198,6 +1202,238 @@ public int ItemPanel_Handler(Menu menu, MenuAction action, int param1, int param
 	}
 }
 
+void ConfirmBuy(int client, int item_id)
+{
+	Panel panel = ItemManager_ConfirmItemPanelInfo(client, item_id, Menu_Buy, true);
+	if (panel != null)
```

Comment:
Maybe there are more elegant way to do this, instead this additional check?
Try refactor `ItemManager_ConfirmItemPanelInfo` function

Reply:
nah dude I gonna keep it in that way

Reply:
```
if (panel == null)
{
	return;
}
```

Because too many nesting